rosdep/python.yaml

```diff
@@ -8519,6 +8535,18 @@ python3-weasyprint-pip:
   ubuntu:
     pip:
       packages: [weasyprint]
+python3-webargs:
+  arch: [python-webargs]
+  debian:
+    pip:
+      packages: [webargs]
+    sid: [python3-webargs]
+  nixos: [python39Packages.webargs]
+  ubuntu:
+    impish: [python3-webargs]
+    jammy: [python3-webargs]
+    pip:
```

Comment:
This needs to target bionic and focal explicitly.

Reply:
resolved in bf1169b2bc4b36c3c5fb44d670d279a9a62bee5d